_data/entertainment.yml

```diff
@@ -30,6 +30,19 @@ websites:
       twitter: Crunchyroll
       facebook: Crunchyroll
 
+    - name: Elven Tower
+      url: http://www.elventower.com/
+      img: elventower.png
+      twitter: elventower
+      facebook: Elven Tower
+      region: na
+      country: MX
+      city: Le
```

Comment:
Is Le the actual city name?

Reply:
yea, not sure about that one, but we're not using that tag yet